proto/viam/app/data/v1/data.proto

```diff
@@ -75,7 +75,7 @@ message Filter {
   string part_name = 8;
   string part_id = 9;
   repeated string location_ids = 10;
-  repeated string org_ids = 11;
+  repeated string organization_ids = 11;
```

Comment:
The binary encoding is based on tag number and type, so this will not affect encoding/compability. This will break the JSON representation, which I think is ok.